content/en/docs/Policy Reports/examples.md

```diff
@@ -131,141 +123,20 @@ summary:
-
-The ClusterPolicyReport, when inspected, has the same structure as the PolicyReport object and contains entries in the `results` and `summary` objects with the outcomes of a policy audit.
-
-```yaml
-results:
-- message: 'validation error: The label `thisshouldntexist` is required. rule check-for-labels-on-namespace
-    failed at path /metadata/labels/thisshouldntexist/'
-  policy: require-ns-labels
-  resources:
-  - apiVersion: v1
-    kind: Namespace
-    name: kube-node-lease
-    uid: 06e5056f-76a3-461a-8d45-2793b8bd5bbc
-  result: fail
-  rule: check-for-labels-on-namespace
-  scored: true
-  source: kyverno
-  timestamp:
-    nanos: 0
-    seconds: 1666098654
-- message: 'validation error: The label `thisshouldntexist` is required. rule check-for-labels-on-namespace
-    failed at path /metadata/labels/thisshouldntexist/'
-  policy: require-ns-labels
-  resources:
-  - apiVersion: v1
-    kind: Namespace
-    name: default
-    uid: 4ffe22fd-0927-4ed1-8b04-50ca7ed58626
-  result: fail
-  rule: check-for-labels-on-namespace
-  scored: true
-  source: kyverno
-  timestamp:
-    nanos: 0
-    seconds: 1666098654
-- message: 'validation error: The label `thisshouldntexist` is required. rule check-for-labels-on-namespace
-    failed at path /metadata/labels/thisshouldntexist/'
-  policy: require-ns-labels
-  resources:
-  - apiVersion: v1
-    kind: Namespace
-    name: kyverno
-    uid: 5d87cd66-ce30-4abc-b863-f3b97715a5f1
-  result: fail
-  rule: check-for-labels-on-namespace
-  scored: true
-  source: kyverno
-  timestamp:
-    nanos: 0
-    seconds: 1666098654
-- message: 'validation error: The label `thisshouldntexist` is required. rule check-for-labels-on-namespace
-    failed at path /metadata/labels/thisshouldntexist/'
-  policy: require-ns-labels
-  resources:
-  - apiVersion: v1
-    kind: Namespace
-    name: kube-public
-    uid: c077ee71-435b-4921-9e05-8751fee71b64
-  result: fail
-  rule: check-for-labels-on-namespace
-  scored: true
-  source: kyverno
-  timestamp:
-    nanos: 0
-    seconds: 1666098654
-- message: 'validation error: The label `thisshouldntexist` is required. rule check-for-labels-on-namespace
-    failed at path /metadata/labels/thisshouldntexist/'
-  policy: require-ns-labels
-  resources:
-  - apiVersion: v1
-    kind: Namespace
-    name: kube-system
-    uid: e63fabde-b572-4b07-b899-b2230f4eac69
-  result: fail
-  rule: check-for-labels-on-namespace
-  scored: true
-  source: kyverno
-  timestamp:
-    nanos: 0
-    seconds: 1666098654
-summary:
-  error: 0
-  fail: 5
-  pass: 0
-  skip: 0
-  warn: 0
-```
+{{% alert title="Note" color="info" %}}
+For a cluster level resource, a `ClusterPolicyReport` would have been created at cluster level instead of a namespaced `PolicyReport`.
+{{% /alert %}}
```

Comment:
This is best collapsed into the same tip as above.

Reply:
Fixed